src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/UnnecessarySemicolonAfterTypeMemberDeclarationCheck.java

```diff
@@ -29,6 +29,8 @@
 /**
  * <p>
  * Checks if unnecessary semicolon is used after type member declaration.
+ * Note: this check is not applicable to empty statements (unnecessary semicolons inside methods or
+ * init blocks), {@code EmptyStatementCheck} is responsible for it.
```

Comment:
Please make a link to the check.

Reply:
Can you show an example how this link should look like?
If in javadoc it is `{@link}` how it should look in xml?

Reply:
https://github.com/checkstyle/checkstyle/blob/master/src/main/java/com/puppycrawl/tools/checkstyle/checks/modifier/ClassMemberImpliedModifierCheck.java#L34
https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/config_modifier.xml#L33

Reply:
done